lib/src/cloud_firestore_mocks_base.dart

```diff
@@ -106,8 +108,18 @@ class MockFirestoreInstance extends Mock implements Firestore {
   }
 
   String dump() {
+    final copy = deepCopy(_root);
+    for (var doc in _docsData.entries) {
+      var out = getSubpath(copy, doc.key);
```

Comment:
Can you add intermediate variables to make it more readable? Like `final docPath = doc.key`, `final documentData = doc.value`, and rename `out` by something like `parentCollection`.

Reply:
You can also make doc, entry and out `final`.

Reply:
Done!